fuzzing/fuzzer_base58.c

```diff
@@ -0,0 +1,15 @@
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+
+#include "parser.h"
+#include "base58.h"
+#include "bip32.h"
+#include "qrcodegen.h"
```

Comment:
```suggestion
#include "base58.h"
```

Reply:
Done. And I also removed unused standard headers.